hack/.staticcheck_failures

```diff
@@ -226,12 +226,10 @@ vendor/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1/helper
 vendor/k8s.io/kube-aggregator/pkg/controllers/autoregister
 vendor/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator
 vendor/k8s.io/kubectl/pkg/cmd/annotate
-vendor/k8s.io/kubectl/pkg/cmd/apply
```

Comment:
That's always good when we can remove some of these (looks like we shouldn't have added them in the first place ...)